app/assets/javascripts/initializers/initializeLocalStorageRender.js

```diff
@@ -8,6 +8,7 @@ function initializeLocalStorageRender() {
       initializeBaseUserData();
       initializeReadingListIcons();
       initializeAllFollowButts();
+      initializeUserFollowButts();
```

Comment:
You can replace `'use strict';` with `/* global initializeUserFollowButts */`. The `'use strict';` is not required and the global comment indicates to eslint that `initializeUserFollowButts` is already defined somewhere else globally.  This should get rid of one of the code climate issues.